COMMUNITY_CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,40 @@
+# Esri Open Source Community Code of Conduct
+
+## Purpose
+
+At Esri, we believe geography can make the world a better place. In adopting this Community Code of Conduct, we as Esri employees are committed to fostering a welcoming environment for collaboration, creativity, and the free exchange of ideas for developing, maintaining, and using open source GIS software. We aim to empower all participants to actively engage and help build a friendly and safe Esri open source developer community on GitHub.
+
+Whether publicly or privately, and whether in-person or online, we expect all members of this community to interact both professionally and without harassment toward others, regardless of race, color, creed, gender, gender identity, religion, marital status, domestic partner status, genetic information, age, national origin or ancestry, military or veteran status, sexual orientation, or either physical or mental disability.
+
+We welcome all contributions that can be submitted under the [Apache 2.0 license](http://www.apache.org/licenses/LICENSE-2.0).
+
+
+## GitHub Expectations
+
+- Make Pull Requests if you have code you want to discuss and merge
+- Use GitHub Issues for general discussions and project-specific discussion and feedback
+- Use [Esri Support](https://developers.arcgis.com/support/) to report API/SDK specific bugs and feature requests; Please don't pester our developers :)
+-  If you cannot come to agreement with Esri Employee decisions, you are always free to fork the project!
+
+See [Contributing to Esri Open Source Projects](https://github.com/Esri/contributing/blob/master/README.md) for more details on copyright, licensing, and using GitHub.
+
+## Community Expectations
+- Feel free to share your ideas, but also listen to others
+- To be professional, kind, and considerate in technical discussions and disagreements
+- Be respectful of personal boundaries and preferences
+- Refrain from demeaning, harassing, aggressive, or intimidating communications and behavior
+- Be respectful of all project contributors and maintainers
+- Report inappropriate behavior
+- Contributors must have the legal right to contribute source code and content under the Apache 2.0 license
+
+## Inappropriate behavior—harassment, aggression, and intimidation
```

Comment:
I don't think we should limit this section heading to "harassment, aggression, and intimidation" as your second paragraph points out. I think simply "Inappropriate behavior" is fine.